src/components/add-account-dialog/add-account-dialog.js

```diff
@@ -10,8 +10,11 @@ export class AddAccountDialogComponent {
     this.props = props;
```

Comment:
please rename component to just `account-dialog` (as we have for tags and transactions)